### PR TITLE
Added ResNext model to ModelZoo

### PR DESCRIPTION
**Pull request by TechNoLog1c**
Added the ResNext [https://arxiv.org/abs/1611.05431] model to the model zoo along with adding the source to tested_models.md 
Exporting works for Caffe, but not Keras and Tensorflow 
screenshot_20181125_231246

### CODE REVIEW

File: docker/dev/reactjs/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
```
Comment (Ram81) [inline docker/dev/reactjs/Dockerfile]: Please revert this

**Comment by TechNoLog1c** [comment]
Fixed the Docker image and dev.sample.py

**Comment by TechNoLog1c** [comment]
Fixed the dockerfile, will see as to why I needed nodejs-legacy and possibly make another PR.

File: docker/prod/reactjs/Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ FROM ubuntu:16.04 as react
 
 RUN apt-get update -qq && apt-get install -y build-essential git curl libfontconfig
 
-RUN apt-get install nodejs-legacy -y
+RUN apt-get install nodejs -y
```
Comment (Ram81) [inline docker/prod/reactjs/Dockerfile]: Why are these changes still here?

File: docker/dev/reactjs/Dockerfile [hunk 3]
```diff
@@ -2,7 +2,7 @@ FROM ubuntu:16.04
 
 RUN apt-get update -qq && apt-get install -y build-essential git curl libfontconfig
 
-RUN apt-get install nodejs-legacy -y
+RUN apt-get install nodejs -y
```
Comment (Ram81) [inline docker/dev/reactjs/Dockerfile]: Why are these changes still here?

File: docker/prod/reactjs/Dockerfile (hunk 2 above)
Comment (TechNoLog1c) [reply docker/prod/reactjs/Dockerfile]: Should be fixed now for sure.

**Comment by TechNoLog1c** [comment]
Added a thumbnail image for ResNext, and also I noticed there wasn't one for ZFnet one either, so I added one as well.

**Comment by TechNoLog1c** [comment]
Fixed the image issues, the thumbnails look fine and aligned to me in the model zoo.

**Comment by TechNoLog1c** [comment]
Fixed the thumbnail size of ZFNet

**Pull request closed without merging**